### PR TITLE
fix(lint): W0611 (unused imports)

### PR DESCRIPTION
**Pull request by feman323**
Removed the unused imports from the files config.py, encfstoos.py and passwd.py. 

This reduces the number of unused imports (W0611) from 28 to 23.

### CODE REVIEW

**Review by buhtz — APPROVED** [review]
Hello Manuel,
On behalf of the team, thank you for your contribution and taking
the time to improve Back In Time. We appreciate it.

Your PR looks fine. It will get merged after a cool off period. To find out how we usually tread PRs please read ["What happens after you opened a Pull Request (PR)?"](https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#what-happens-after-you-opened-a-pull-request-PR).

Best,
Christian

**Comment by buhtz** [comment]
Hello Manual,
you asked if you can add more modifications.

My stomach tells me ~10 files are OK. 25 errors are Ok. Fix them all if you want.

You don't have to create a new PR for this. Just work further on your existing branch. Commit and push. This PR will be updated automatically.

Best,
Christian

File: common/doc-dev/conf.py
Comment (feman323) [inline common/doc-dev/conf.py]: I'm not sure how to deal with this . Does this mean the unused import needs to stay for some reason?

Comment (buhtz) [reply common/doc-dev/conf.py]: How did you find that line? You shouldn't get an error about it because of the comment `pylint: disable-next..`.

Anway you can delete this `import` and the related comment lines.

I am not 100% sure about it. But I checked the git history and couldn't find a good reason to keep that import. I was also able to build the doc via Sphinx without that import.

File: common/qt_probing.py
Comment (buhtz) [inline common/qt_probing.py:104]: Don't delete this line. It is inside a try-block. The purpose here is to provoke an ImportError. That probing script is a workaround from "aryoda".

Add this comment line before that import to disable pylint for that line:

    # pylint: disable-next=unused-import

**Review by buhtz — CHANGES_REQUESTED** [review]
Nice. Just added some minor suggestions and modifications.

**Comment by buhtz** [comment]
Hello Manuel,
we plan to release a new BIT version (might be 1.5.0) in the end of July to make it hit the upcoming release of Ubuntu 24.10 (Oracular Oriole) in October this year. Ubuntus [Debian Import Freeze](https://discourse.ubuntu.com/t/oracular-oriole-release-schedule/36460?u=d0od) is at August 15th. So it is an important release.

To my understanding, knowledge and experience I see no real problem or real risk with cleaning up the unused imports. But it is also my experience that I am often wrong with assumptions like this. 😆 Because of that I would like to put your PR on hold until the upcoming release is out and merge it after that.

Best,
Christian

**Comment by feman323** [comment]
> Hello Manuel, we plan to release a new BIT version (might be 1.5.0) in the end of July to make it hit the upcoming release of Ubuntu 24.10 (Oracular Oriole) in October this year. Ubuntus [Debian Import Freeze](https://discourse.ubuntu.com/t/oracular-oriole-release-schedule/36460?u=d0od) is at August 15th. So it is an important release.
> 
> To my understanding, knowledge and experience I see no real problem or real risk with cleaning up the unused imports. But it is also my experience that I am often wrong with assumptions like this. 😆 Because of that I would like to put your PR on hold until the upcoming release is out and merge it after that.
> 
> Best, Christian

Hi Christian,

sure I totally understand that. This is not just your experience :grin: 

Regards,

Manuel

File: common/qt_probing.py
Comment (feman323) [reply common/qt_probing.py:104]: Done

File: common/doc-dev/conf.py
Comment (feman323) [reply common/doc-dev/conf.py]: The "pylint: disable-next..." wasn't there, I added it in commit 67506d3. Thats how I found the line.

File: common/qt_probing.py
Comment (buhtz) [inline common/qt_probing.py:104]:
```suggestion
        # pylint: disable-next=unused-import
```

**Review by buhtz — CHANGES_REQUESTED** [review]

File: common/qt_probing.py
Comment (feman323) [reply common/qt_probing.py:104]: Hi Christian,

I don't understand this suggested change. What's the point of this additional indentation?

Regards,

Manuel

Comment (buhtz) [reply common/qt_probing.py:104]: Yes, that makes no sense. Thank you for looking carefully. :+1:

**Review by buhtz — APPROVED** [review]

**Pull request merged**
